github/repos.go

```diff
@@ -176,8 +176,10 @@ func (s *RepositoriesService) List(ctx context.Context, user string, opt *Reposi
 		return nil, nil, err
 	}
 
-	// TODO: remove custom Accept header when license support fully launches
-	req.Header.Set("Accept", mediaTypeLicensesPreview)
+	// TODO: remove custom Accept headers when APIs fully launch.
+	acceptHeader := []string{mediaTypeLicensesPreview, mediaTypeTopicsPreview}
+
```

Comment:
Also, call it `acceptHeaders` since it's more than one. And it's consistent with existing `acceptHeaders` in `RepositoriesService.Get`.